### PR TITLE
add new version support: prep for release automation

### PR DESCRIPTION
**Pull request by mmulholla**
Add support for reading the chart verifier version from a json file in preparation for release automation.

### CODE REVIEW

File: cmd/version.go [hunk 1]
```diff
@@ -1 +1,68 @@
...
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+var Version = "0.0.0"
```
Comment (zonggen) [inline cmd/version.go:12]: Minor: Is there a reason this is exported?

File: cmd/version_test.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/mod/semver"
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
+
+	t.Run("Check Version is set.", func(t *testing.T) {
+		fmt.Println(fmt.Sprintf("Version is %s", Version))
```
Comment (zonggen) [inline cmd/version_test.go:13]: Minor: linter suggests: `fmt.Printf("Version is %s\n", version)`

File: cmd/version.go (hunk 1 above)
Comment (mmulholla) [reply cmd/version.go:12]: Probably not while it is only used in verify.go although it is the source of truth for the version so like it being available.

File: cmd/version_test.go (hunk 2 above)
Comment (mmulholla) [reply cmd/version_test.go:13]: Not a fan of "\n" in the text, prefer Println to take care of that.

**Review by dperaza4dustbit — APPROVED** [review]

**Review by zonggen — APPROVED** [review]

**Pull request merged**